.github/workflows/unit-tests.yml

```
@@ -12,46 +12,25 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - '3.10'
-          - '3.11'
-          - '3.12'
-          - '3.13'
-        poetry-version: ["1.8.3"]
+          - "3.10"
+          - "3.11"
+          - "3.12"
+          - "3.13"
```

```suggestion
          - "3.13"
          - "3.14"
```

3.14 is released. probably not a bad idea to include it